coderd/database/dbmem/dbmem.go

```diff
@@ -7713,7 +7753,15 @@ func (q *FakeQuerier) GetAuthorizedWorkspaces(ctx context.Context, arg database.
 		w1 := workspaces[i]
 		w2 := workspaces[j]
 
-		// Order by: running first
+		// Order by: favorite first
+		if arg.OrderByFavorite == w1.OwnerID && w1.Favorite {
```

Comment:
If both are favorite, this might not produce the desired order. 🤔

Reply:
Yeah that's possible, but as it's just `dbmem` I'm going to defer caring about it until it makes a test fail ;-)